lib/train/transports/local.rb

```diff
@@ -199,10 +199,16 @@ def start_pipe_server(pipe_name)
               $scriptBlock = $ExecutionContext.InvokeCommand.NewScriptBlock($command)
               try {
                 $stdout = & $scriptBlock | Out-String
-                $result = @{ 'stdout' = $stdout ; 'stderr' = ''; 'exitstatus' = 0 }
+                $exit_code = $LastExitCode
+                if ($exit_code -eq $null)
+                {
+                  $exit_code = 0
```

Comment:
This treats an unset `$LastExitCode` as `0`.
This is fiddling the result, but it preserves the previous behaviour where we set the exit_code to 0 if nothing went wrong.
I could _not_ do this, but it might break workflow if any current users are testing that 0 is returned for their command.